packages/cli/bin/download-connector-list.js

```diff
@@ -0,0 +1,56 @@
+const https = require('https');
+const fs = require('fs');
+const path = require('path');
+const util = require('util');
+
+const readFileAsync = util.promisify(fs.readFile);
+const writeFileAsync = util.promisify(fs.writeFile);
+
+const dest = path.resolve('generators/datasource/connectors.json');
+const URL =
+  'https://raw.githubusercontent.com/strongloop/loopback-workspace/master/available-connectors.json';
+
+/**
+ * Function to dowload the list of available connectors from loopback-workspace
+ * so the list only has to be maintained in one place.
+ */
+async function download() {
+  var file = fs.createWriteStream(dest);
+  var request = https
+    .get(URL, function(response) {
```

Comment:
Hmm, why don't we keep a local copy with `@loopback/cli`?

Reply:
I realize that it's used as part of the build.

Reply:
Yep. Build pulls in the latest from `loopback-workspace` as we didn't want to maintain two lists of connectors & didn't want to make `loopback-workspace` a dependency.